### PR TITLE
Improving CodeCoverage adding a test on EIT. Removing a dupplicated gain.

### PR DESCRIPTION
**Pull request by eolivi**

### CODE REVIEW

**Comment by agramfort** [comment]
@eolivi I see a segfault in EITSM-Head1 in appveyor

File: OpenMEEG/src/TestHead.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ function(TESTHEAD HEADNUM)
     set(DIPPOS                 ${MODELBASE}.dip)
     set(DIPPOS-SKULLSCALP      ${MODELBASE}-skullscalp.dip)
     set(PATCHES                ${MODELBASE}.patches)
+    set(EITPATCHES             ${MODELBASE}.EITpatches)
```
Comment (agramfort) [inline OpenMEEG/src/TestHead.txt]: I would rename to : ${MODELBASE}-EIT.patches
unless .patches is different as the file format is different.

Comment (eolivi) [inline OpenMEEG/src/TestHead.txt]: a EIT patch has a surfacic extension. hence the 4th numeric column, this is the radius of the patch.
so do I rename it ?

File: OpenMEEG/libs/OpenMEEG/include/gain.h [hunk 2]
```diff
@@ -213,13 +213,4 @@ namespace OpenMEEG {
         }
         ~GainStimInternalPot () {};
     };
-
-   class GainEITInternalPot : public Matrix {
-   public:
```
Comment (eolivi) [inline OpenMEEG/libs/OpenMEEG/include/gain.h]: as discussed with Theo, I should keep the GainEITInternalPot, and remove the GainStimInternalPot instead.
(this is not mentioned in the doc. only used once in openmeeg_sample_data)

**Comment by papadop** [comment]
Merged. Thanks.

**Pull request merged**